notes-apps-ogechukwu-okoli.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+class Note:#base class
+    '''
+    It initializes new notes
```

Comment:
While your comments are helpful, a few could be more descriptive. For instance, instead of “It initializes new notes” in the Note class, a quick note on its role (like “A base class to store note details”) might make it even clearer.